### PR TITLE
{bio}[foss/2019b] arcasHLA v0.2.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node380.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/835c5910e8f9e8753b549f9d42c4f0c9 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/9e4e406770a8e007a1dfe430b6d1194f for a full test report.

File: easybuild/easyconfigs/a/arcasHLA/arcasHLA-0.2.0-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    ('Biopython', '1.75', versionsuffix),
+    ('BEDTools', '2.29.2'),
+    ('kallisto', '0.46.1'),
+    ('SAMtools', '1.10'),
+    ('pigz', '2.4'),
+    ('git', '2.23.0', '-nodocs'),
+    ('git-lfs', '2.11.0', '', True),
+]
+
+# Download and install the referende database (1.8 GB)
```
Comment (verdurin) [inline easybuild/easyconfigs/a/arcasHLA/arcasHLA-0.2.0-foss-2019b-Python-3.7.4.eb:30]: @lexming while I don't wish to hold this up, please fix the typo here

Comment (lexming) [reply easybuild/easyconfigs/a/arcasHLA/arcasHLA-0.2.0-foss-2019b-Python-3.7.4.eb]: @verdurin Good catch. Fixed :+1: 
BTW, it's not a problem for me if you would commit this type of fixes to my PRs, I really don't mind :slightly_smiling_face:

**Comment by lexming** [comment]
@verdurin there are two commits because I messed up the version of SAMtools (the problem of local customizations), but those are only a fix for the typo

**Comment by verdurin** [comment]
@lexming thanks for clarifying, was a bit confused about that one.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node379.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/8a31823df8aaf9ae92b4feb09b7ddcd9 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/9ede232b9956bd52ee160a0ebd82fdbb for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/42c6517965dab9669674c301aae2e058 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @lexming!

**Pull request merged**